flink-runtime-web/web-dashboard/src/app/pages/job/overview/flamegraph/job-overview-drawer-flamegraph.component.html

```diff
@@ -41,6 +41,6 @@
   <nz-spin *ngIf="isLoading"></nz-spin>
 </div>
 
-
-
-
+<ng-template #notAvailable>
+  <span>Flame graph is currently unavailable</span>
```

Comment:
I'm thinking of your comment about https://issues.apache.org/jira/browse/FLINK-22527 - should this be solved here instead of inside https://github.com/apache/flink/pull/16352?

Reply:
In that case, since (afaik) we can't distinguish between the Flamegraphs being disabled vs. us just waiting for the first results to appear, we'd have to provide a generic text like
> Flame graph currently unavailable
> We are either waiting for the first samples to create a flame graph or this feature is currently disabled (enable it by setting [`rest.flamegraph.enabled: true`](https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/deployment/config/#rest-flamegraph-enabled))

If we can actually distinguish the two, then splitting these scenarios up into two different messages would be even better.

Reply:
Pls consider my solution in FLINK-23102. Pls comment below if there is anything else for me to do.